modules/cats/src/main/tut/README.md

```diff
@@ -0,0 +1,41 @@
+# Cats module for PureConfig
+
+Adds support for selected [cats](http://typelevel.org/cats/) classes to PureConfig.
+
+## Add pureconfig-cats to your project
+
+In addition to [core pureconfig](https://github.com/melrief/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-cats" % "0.7.1"
+```
+
+## Example
+
+To load a `NonEmptyList[Int]` into a configuration, we need a class to hold our configuration:
```

Comment:
I'm worried that this may give users the impression that they always have to wrap `NonEmptyList` into a class, even if the list is already part of a bigger hierarchy. The only reason why `parseString("[1,2,3]")`doesn't work is because a `Config` (returned by `parseString`) can only hold a config object. Maybe we can explain this here?